libqtile/layout/floating.py

```diff
@@ -28,20 +28,18 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from libqtile.layout.base import Layout
+from collections import defaultdict
 
-DEFAULT_FLOAT_WM_TYPES = set([
-    'utility',
-    'notification',
-    'toolbar',
-    'splash',
-    'dialog',
-])
+from libqtile.layout.base import Layout
+from libqtile.log_utils import logger
 
-DEFAULT_FLOAT_RULES = [
-    {"role": "About"},
-    {"wmclass": "file_progress"},
-]
+# sensible floating defaults like in i3
+# https://github.com/i3/i3/blob/ae757c6848b49d7a0423973a39791ba6eca29308/src/manage.c#L449
+# except `A__NET_WM_STATE_MODAL` windows (couldn't figure this one out)
```

Comment:
Isn't thinks just "wmclass": "__NET_WM_STATE_MODAL"?

Reply:
Or maybe we do some fancy processing, so "wmclass": "modal"?

Reply:
I am not sure about this. There is a method `get_net_wm_state` in here https://github.com/qtile/qtile/blob/master/libqtile/backend/x11/xcbq.py#L603 , but it returns a tuple.